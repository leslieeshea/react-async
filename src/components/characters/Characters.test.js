import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('Characters component', () => {
  it('renders a list of characters', () => {
    const characters = [
      {
        name: 'Morty Smith',
        status: 'Alive',
        species: 'Human',
        image: '/image'
      },
      {
        name: 'Summer Smith',
        status: 'Alive',
        species: 'Human',
        image: '/image'
      }
    ];

    const wrapper = shallow(<Characters characters={characters} />);
    expect(wrapper).toMatchSnapshot();
  });
});
