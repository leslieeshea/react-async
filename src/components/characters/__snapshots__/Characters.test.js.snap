// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Characters component renders a list of characters 1`] = `
<ul
  className="Characters"
>
  <li
    key="0"
  >
    <Character
      character={
        Object {
          "image": "/image",
          "name": "Morty Smith",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Character
      character={
        Object {
          "image": "/image",
          "name": "Summer Smith",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
</ul>
`;
